config/planner.yaml

```diff
@@ -0,0 +1,56 @@
+controller_frequency: 5.0
+recovery_behaviour_enabled: true
+
+NavfnROS:
```

Comment:
I might be nitpicking here, but @kruusamae do you not think that it's better to separate the global planner and local planner yaml files? Or if we want to keep it like this, might make more sense to name the yaml file as "planners".
If we segregate the global and local planners' yaml files, it would be a little bit neater in case one wants to set some other parameters for each of the planners.

Reply:
i agree, it seems like an obvious thing to do - separating `planner.yaml` into `planner_local.yaml` and `planner_global.yaml`. it would follow the same logic as `costmap_*.yaml` files do.

@Houman-HM what do you think would there need to be a `planner_common.yaml` or in which file exactly would the following two lines need to go?
```
controller_frequency: 5.0
recovery_behaviour_enabled: true
```

@mmaao could you please split `planner.yaml` into multiple files and modify the launch-file accordingly?

Reply:
@mmaao you can create three different yaml files as local_planner_params, global_planner_params, and move_base params, which would include the related parameters for the local_planner, global_planner, and move_base (aka common parameters), respectively.
You can take look a [this](https://github.com/turtlebot/turtlebot_apps/tree/indigo/turtlebot_navigation/param) repo and see on how they've done it.